Triangulation_3/test/Triangulation_3/CMakeLists.txt

```diff
@@ -29,6 +29,9 @@ create_single_source_cgal_program("test_static_filters.cpp")
 create_single_source_cgal_program("test_triangulation_3.cpp")
 create_single_source_cgal_program("test_io_triangulation_3.cpp")
 create_single_source_cgal_program("test_triangulation_serialization_3.cpp")
+create_single_source_cgal_program("test_dt_deterministic_3.cpp")
```

Comment:
@lrineau  The test_dt_determinisitic_3,cpp fails in the testsuite.  Can you please investigate.  Is the test bugy, should it be removed, and IS Dt_3 determinisitic or not.